xml/System.Numerics/INumberBase`1.xml

```diff
@@ -782,6 +782,8 @@ A return value of `false` does not imply that <xref:System.Numerics.INumberBase%
 
 This function returns `true` for a complex number `a + bi` where `b` is zero.
 
+This function also returns `true` for positive and negative infinity.
```

Comment:
I'm not sure this is the "right" wording.

Technically speaking `infinity` is a `real number` and lives on the `extended real number line`. This is the number line that the IEEE 754 specification follows and which is used in many higher level mathematical domains to allow correct modeling of various concepts.

Trying to define it as "this also returns true for ..." makes it instead sound like `infinity` is not a real number, which is then misleading.

I'd rather see the remark elaborate something like:
> This function checks values against the `extended real number line` which includes `positive infinity` and `negative infinity`.

Reply:
That seems way clearer. I've updated the docs.